src/saturn_engine/core/topic.py

```diff
@@ -41,9 +44,14 @@ def __init__(
             tags: dict[str, str] = None,  # type: ignore[assignment]
             config: dict[str, dict[str, t.Optional[t.Any]]] = None,  # type: ignore[assignment]
             metadata: dict[str, dict[str, t.Optional[t.Any]]] = None,  # type: ignore[assignment]
+            expire_after: t.Optional[datetime.timedelta | int] = None,
         ) -> None:
             ...
 
+    def __post_init__(self) -> None:
+        if isinstance(self.expire_after, datetime.timedelta):
+            self.expire_after = self.expire_after.seconds
```

Comment:
@isra17 I think it's valuable to set the expiration with a timedelta, but we need datetime cannot be serialize.

Reply:
done.